src/pghstore/_native.py

```diff
@@ -199,7 +196,7 @@ def load(file, encoding='utf-8'):
     if not callable(read):
         raise TypeError('file must be a readable file object that implements '
                         'read() method')
-    return load(read(), encoding=encoding)
+    return loads(read(), encoding=encoding)
```

Comment:
I think it wasn't totally broken previously because we've never not used the C extension.